packages/nimble-components/src/card/styles.ts

```diff
@@ -21,4 +22,12 @@ export const styles = css`
         color: ${bodyFontColor};
         background-color: ${sectionBackgroundColor};
     }
+
+    ::slotted([slot='title']) {
+        font: ${titleFont};
```

Comment:
I'm not sure if it's what we want or not, but I'll point out that the title currently wraps when it gets too long. In some of our components, we limit the title to only being one line and show ellipsis if it gets too long. Since we get this current behavior for free & the card is incubating, I personally don't have an issue leaving the style as-is. However, it would be good to get an answer from designers on this before we consider the styles final.

![image](https://github.com/ni/nimble/assets/20542556/a07f63fe-9608-4963-866d-5a904d5992db)

Reply:
I brought this up with our interaction designer, and she's going to discuss it with her group. I'll add a note to my user story for finalizing the styling to include this if we decide we need to.